packages/admin/admin-theme/src/componentsTheme/MuiListItemIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { mergeOverrideStyles } from "../utils/mergeOverrideStyles";
+import { GetMuiComponentTheme } from "./getComponentsTheme";
+
+export const getMuiListItemIcon: GetMuiComponentTheme<"MuiListItemIcon"> = (component) => ({
+    ...component,
+    defaultProps: {
+        ...component?.defaultProps,
+    },
+    styleOverrides: mergeOverrideStyles<"MuiListItemIcon">(component?.styleOverrides, {
+        root: {
+            minWidth: "unset !important",
```

Comment:
Please override the `min-width` using a more precise selector to remove the `!important`. 
As you can see in the dev-tools, MUI sets the `min-width` from `MuiMenuItem` onto `MuiListItemIcon`, so you should be able to override this by setting the style on `.MuiListItemIcon-root` inside `getMuiMenuItem`. 

Also I'd slightly prefer the actual initial value of `minWidth` instead of using `unset` as this makes it a bit easier to see what style is actually set. 

<img width="259" alt="MinWidth in the Dev-Tools" src="https://github.com/user-attachments/assets/2871c102-95f0-41b3-859b-a0272c5ea353" />